docs/Token-Swap.md

```diff
@@ -0,0 +1,9 @@
+# Scenario: Token swap
```

Comment:
This is the old `Asset-Exchange.md` file, renamed it to be consistent with the Token Swap nomenclature, changed some words from the first section and removed the second that I couldn't make sense of